Tests/GitHubBranches.tests.ps1

```diff
@@ -100,13 +100,163 @@ try
                 $branchAgain.name | Should -Be $branchName
             }
 
-            It 'Should have the exected type and addititional properties' {
+            It 'Should have the expected type and addititional properties' {
                 $branchAgain.PSObject.TypeNames[0] | Should -Be 'GitHub.Branch'
                 $branchAgain.RepositoryUrl | Should -Be $repo.RepositoryUrl
                 $branchAgain.BranchName | Should -Be $branchAgain.name
             }
         }
     }
+
+    Describe 'GitHubBranches\New-GitHubRepositoryBranch' {
+        Context 'When creating a new GitHub repository branch' {
+            BeforeAll {
+                $repoName = [Guid]::NewGuid().Guid
+                $originBranchName = 'master'
+                $newBranchName = 'develop'
+                $newGitHubRepositoryParms = @{
+                    RepositoryName = $repoName
+                    AutoInit = $true
+                }
+
+                $repo = New-GitHubRepository @newGitHubRepositoryParms
+
+                $newGitHubRepositoryBranchParms = @{
+                    OwnerName = $script:ownerName
+                    RepositoryName = $repoName
+                    BranchName = $newBranchName
+                    OriginBranchName = $originBranchName
+                }
+
+                $branch = New-GitHubRepositoryBranch @newGitHubRepositoryBranchParms
+            }
+
+            It 'Should support pipeline input for the uri parameter' {
+                { $repo | New-GitHubRepositoryBranch -BranchName $newBranchName -WhatIf } |
```

Comment:
Based on our previous experience with #259, I'd prefer that pipeline testing be practical and not exercised by `WhatIf` -- here and elsewhere.

Reply:
Done